selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -645,6 +639,34 @@ public void revealResourceByOptionsButton() {
     clickOnOptionsMenuItem(ProjectExplorerOptionsMenuItem.REVEAL_RESOURCE);
   }
 
+  /**
+   * perform the multi-select by Shift key
+   *
+   * @param path is the path to the selected item
+   */
+  public void selectMultiFilesByShiftKey(String path) {
+    Actions actions = actionsFactory.createAction(seleniumWebDriver);
+    actions.keyDown(Keys.SHIFT).perform();
+    selectItem(path);
+    waitItemIsSelected(path);
+    actions.keyUp(Keys.SHIFT).perform();
+  }
+
+  /**
+   * perform the multi-select by Shift key with check selected items
+   *
+   * @param clickItemPath is the path to the selected item
+   * @param selectedItemsPaths is the paths to the each selected items after click with shift button
+   */
+  public void selectMultiFilesByShiftKeyWithCheckMultiselection(
+      String clickItemPath, List<String> selectedItemsPaths) {
+    Actions actions = actionsFactory.createAction(seleniumWebDriver);
+    actions.keyDown(Keys.SHIFT).perform();
```

Comment:
Let's use static import for simplifying constant references similar to how it had been done in other parts of code.